### PR TITLE
add c++ application for EEA token execution logic.  Signed-off-by: reckeyzhang <lei.zhang@iex.ec>

### PR DESCRIPTION
**Pull request by reckeyzhang**

### CODE REVIEW

File: examples/apps/eea_token/execution_logic.h [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+#define ERROR_SHARE_TOKENS                          4
+
+#define ACTIVITY_PARTICIPATION_MEETING              1
+#define ACTIVITY_CHAIR_MEETING                      2
+#define ACTIVITY_CONTRIBUTE_DELIVERABLE             3
+#define ACTIVITY_EDITOR_DELIVERABLE                 4
+#define ACTIVITY_CONTRIBUTE_EEA_PROJECT_REGULAR     5
+#define ACTIVITY_CONTRIBUTE_EEA_PROJECT_MANAGER     6
+#define ACTIVITY_CONTRIBUTE_EEA_PROJECT_SPONSOR     7
+#define ACTIVITY_NUMBER                             7
```
[the remark this answers is not in the document]
Comment (reckeyzhang) [reply examples/apps/eea_token/execution_logic.h]: Actually, ACTIVITY_NUMBER represents the current total number of activities (1-7), we do not name the activity type starting with 0 in order to conform with the activity type indices defined by EEA token spec.

**Comment by danintel** [comment]
You have 3 commits in this PR. Only the 2nd commit of the 3 has the DCO sign off line:
`https://github.com/hyperledger-labs/trusted-compute-framework/pull/38/commits`
See https://github.com/hyperledger-labs/trusted-compute-framework/pull/38/commits

You need to rebase your commits (I would use `git rebase -i HEAD~3`) and add the missing DCO sign off line for the 1st and 3rd commits. I would use the `rewrite` option in the rebase to add the missing DCO sign off line. There are other ways of doing this also.

**Comment by reckeyzhang** [comment]
Hello,

In order to comply the standard procedure, please remove the branch
"eea-token-app", I made a pull request directly from my repo.

Thanks and regards

Lei

**Comment by danintel** [comment]
Lei,

I tried deleting branch “eea-token-app” branch and did not have permission.

Can you try to delete it?  Delete the branch from
 https://github.com/hyperledger-labs/trusted-compute-framework/branches

Dan

**Comment by reckeyzhang** [comment]
Hello,

I tried to delete the branch from
https://github.com/hyperledger-labs/trusted-compute-framework/branches, but
it says "You can't delete this branch", and I do not have the right to do
the modification to the "setting->branch".
I also tried to run "git push --delete origin eea-token-app", but it always
says "remote: error: GH006: Protected branch update failed for
refs/heads/eea-token-app."

Could you please kindly give me any hints on deleting this branch?

Thanks

Lei

**Comment by danintel** [comment]
I cannot delete the branch either. Just leave the branch and I will see if Eugene has authorization.

**Pull request closed without merging**